main.go

```diff
@@ -141,18 +141,25 @@ func registerEHR(server *echo.Echo, config Config) {
 		log.Fatal(err)
 	}
 
-	patientRepository := patients.NewFHIRPatientRepository(patients.Factory{}, config.FHIRServerAddress)
+	fhirClientFactory := fhir.NewFactory(fhir.WithURL(config.FHIR.Server.Address))
+	patientRepository := patients.NewFHIRPatientRepository(patients.Factory{}, fhirClientFactory)
 	transferRepository := transfer.NewSQLiteTransferRepository(sqlDB)
 	orgRegistry := registry.NewOrganizationRegistry(&nodeClient)
 	vcRegistry := registry.NewVerifiableCredentialRegistry(&nodeClient)
-	transferService := transfer.NewTransferService(authService, fhir.NewClient(config.FHIRServerAddress), transferRepository, customerRepository, orgRegistry, vcRegistry)
+	transferService := transfer.NewTransferService(authService, fhirClientFactory, transferRepository, customerRepository, orgRegistry, vcRegistry)
+	tenantInitializer := func(tenant string) error {
+		return fhir.InitializeTenant(config.FHIR.Server.Type, config.FHIR.Server.Address, tenant)
+	}
 
 	if config.LoadTestPatients {
 		allCustomers, err := customerRepository.All()
 		if err != nil {
 			log.Fatal(err)
 		}
 		for _, customer := range allCustomers {
+			if err := tenantInitializer(customer.Id); err != nil {
```

Comment:
The demo is currently able to use new customers without a restart, this would require a restart.